### ISSUE TITLE
Including mixins doesn't work

### ISSUE THREAD

**Issue by shdnx** [pugjs/pug#2394]
Consider the following mixin living in common/central-panel.jade:

```
mixin centralPanel(title)
    h3= title
    block
```

That I'm trying to use in 404.jade:

```
extends common/layout.jade
    
block body
    include common/central-panel.jade
    +centralPanel('Error 404')
        p
            | The page you requested could not be found! We're so sorry!
```

I get the following error:

>     TypeError: .../src/markup/404.jade:5
>         3| block body
>         4|     include common/central-panel.jade
>       > 5|     +centralPanel('Error 404')
>         6|         p
>         7|             | The page you requested could not be found! We're so sorry!
>     
>     Cannot read property 'call' of undefined

Furthermore, the include statement doesn't seem to be allowed on the top-level in 404.jade, causes pug to crash.

I'm using pug 2.0.0-alpha7. It works with jade 1.11.0.

**Comment by TimothyGu** [comment]
Can you try updating Pug to 2.0.0-alpha8? I suspect the error is because of the `.jade` filename suffix, which was not recognized as a Pug file before a8.

**Comment by shdnx** [comment]
@TimothyGu: thanks, it's indeed fixed in 2.0.0-alpha8! The behaviour was odd, because there was no error or warning for trying to include a non-pug file, and using the .jade extension worked fine for 'extends'. Thank you for your work!
